src/main.rs

```diff
@@ -169,9 +165,10 @@ fn main() {
             }
         };
 
+        emit!(VectorStopped);
```

Comment:
Obviously very much new here ... for consistency should the `emit!` for shutdown be where the `info!` is currently?

Reply:
Yea, I thought this was better place, but this clearer.